### ISSUE TITLE
can not open app

### ISSUE THREAD

**Issue by resigafoos** [yep/DeviceSync-for-OS-X#1]
app will not open

**Comment by resigafoos** [comment]
I have White USB cable form back of iMac to iPhone.
I try to sync and getting message on iPhone that NO USB cable is connected. ???
Reboot iPhone and Start over and get same message.

**Comment by yep** [comment]
You said "app will not open" in the first post. Did you get an error message like "the app is from an unidentified developer"? 
If so, please have a look at https://support.apple.com/kb/HT5290
What they say there is that you have to control-click or right-click the app "DeviceSync for OS X" for it to open.

Both apps have to be running to be able "see" and "talk to" each other.

Does this help?

**Comment by resigafoos** [comment]
thanks...but the sync only synced iCloud events. Sync also deleted all other events on computer 
calendar. It also added a second duplicate birthday calendar on computer.

**Comment by yep** [comment]
Thank you for your feedback. To track down the problem that not all calendars are exported I would need your support.

When the sync operation finished, could you please have a look at the output at your iPhone? It should say something like "Exporting calendar with title 'Some Calendar Title'".
Now for the calendar that is not synchronized, what does the next line say? It should be something like "Found x calendar events.".

I know that all other events are deleted. For the moment, please only add and update events on your iPhone. That way, you at least have a local copy of the events on your iMac.

It created a duplicate birthday calendar at my computer, too.

As my time permits I will have to find a better way than that for the above two problems.

**Comment by guay** [comment]
I use an iPhone with iOS 7.0.4 and Mavericks (10.9) on my MacBook Pro. I can start both Apps on both devices. On my MacBook I get the screen:
WARNING: ALL LOCAL CALENDAR DATA WILL BE OVERWRITTEN ON SYNCHRONIZATION!
Plug in USB cable and launch 'DeviceSync for iOS' to start calendar synchronization.
So, i plug in the USB cable, start the app on my iPhone and click on "Sync", but nothing happens. The error message on the iphone is: Can not synchronize calendar. No USB connection.

The USB cable is for sure fully ok, since it works with other apps. What else can I do to make it work?

**Comment by guay** [comment]
Update. I deleted the app on the iPhone and reloaded it again. It worked. But, surprise, it exported only the first 100 Events. I guess, the app is still not fully operational. But anyway, it is one step in the right direction... away from the iCloud bondage.

**Comment by yep** [comment]
>  I guess, the app is still not fully operational. But anyway, it is one step in the right direction...

You are right, the version currently in the AppStore (1.0) was more of a test if Apple allows certain communication (TCP over USB using UNIX sockets) in the AppStore. Without being sure if they accept this, all future work would have be in vain. Now that it's in the store, I will continue adding more feature - as my time allows.

Three days ago I pushed version 1.1 to AppStore review, should be available automatically in the store next week. There it is possible to configure the amount of days in the past in the future you want to sync/copy.

Thinks I already started are contacts, albeit also only one way (iOS to OSX) at the start.
After contacts, I will probably have a look at improving the merging behavior of contacts/calendar entires.

If you have any more feature requests, feel free to open github issues for them.
